client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -223,7 +223,7 @@ export const PatternLibrary = ( {
 					<CategoryGallery
 						title={ translate_not_yet( 'Ship faster, ship more' ) }
 						description={ translate_not_yet(
-							'Choose from a library of beautiful, functional design patterns to build exactly the page you—or your client—need, in no time.'
+							'Choose from a library of beautiful, functional design patterns to build exactly the pages you need—or your client needs—in no time.'
```

Comment:
I was about to suggest adding spaces around the dashes:
```suggestion
							'Choose from a library of beautiful, functional design patterns to build exactly the pages you need — or your client needs — in no time.'
```

However, it seems that's actually the convention defined by the Chicago Manual of Style (CMOS), which we use at Automattic. So all good!